staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go

```diff
@@ -79,7 +80,7 @@ type Interface interface {
 	Create(ler LeaderElectionRecord) error
```

Comment:
I'd recommend at least expanding at least Get and Create while we're here. No reason not to. 

```
type Interface interface {
	Get(ctx context.Context) (*LeaderElectionRecord, []byte, error)
	Create(ctx context.Context, ler LeaderElectionRecord) error
}
```